samples/config/config.sample

```diff
@@ -71,3 +71,34 @@
 # Keys:
 #   camtopic
 #      Gazebo topic where camera images are published.
+#
+#
+# Section [imgcap]:
+#
+# Keys:
+#   width
+#       Width of the image to be captured in pixels.
+#       Default: Width of the camera sensor.
+#       Possible Values: Dependent on camera sensor
+#
+#   height
+#       Height of the image to be captured in pixels.
+#       Default: Height of the camera sensor.
+#       Possible Values: Dependent on camera sensor
+#
+#   format
+#       Image format
+#       Default: 2 (JPEG)
+#       Possible Values:
+#            1 - Raw Format  (IMAGE_FILE_RAW)
+#            2 - JPEG Format (IMAGE_FILE_JPEG)
```

Comment:
If this is the only value supported then I'd only display this one. Add the others later.